tests/data_test.py

```diff
@@ -166,6 +166,23 @@ def test_ecoacoustics_dm(root: Path):
     assert test_sample.x.size()[3] == dm.dims[2]
 
 
+@pytest.mark.slow
+def test_ecoacoustics_dm_batch_multi_label(root: Path) -> None:
```

Comment:
Check that this PR works with varying batch_sizes. You can parametrize the test with the below, and the pass this as an arg to the dataloader

```suggestion
@pytest.mark.slow
@pytest.mark.parametrize("batch_size", [1, 2, 8,])
def test_ecoacoustics_dm_batch_multi_label(root: Path, batch_size: int) -> None:
```